src/options.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * These are all options available to Ars Arsenal
+ * @flow
+ */
+
+import { type ID, type Record } from './record'
+
+export type ArsMode = 'gallery' | 'table'
+
+export type ArsColumn = $Keys<Record> | 'preview'
+
+export interface ArsOptions {
+  // The base URL for API interaction
+  url: string;
+  // Define how the endpoint url is constructed
+  makeURL: (string, ID) => string;
+  // Define how the search query string is built
+  makeQuery: string => string;
+  // Configure the root element's HTML attributes
+  rootAttributes: Object;
+  // Format errors before they are sent as a "string" value
+  // to the component
+  onError: (error: *) => string;
+  // Format the response, useful if you do not control the
+  // JSON response from your endpoint
+  onFetch: (respose: *) => *;
+  // Whenever a new item is picked, this event is triggered
+  // When using multiselect: true, this is an array of values
+  onChange: (*) => void;
+  // Are multiple selections possible?
+  multiselect: boolean;
+  // The noun used for selection, i.e. "Pick a photo"
```

Comment:
Would I be entering "Pick a photo" or "photo" as the option? I think it's just the noun, but this was a little confusing to me.

Reply:
Cool. This is definitely confusing. Great catch!

Reply:
Took a pass at this in acd8926